debug/package.json

```diff
@@ -4,8 +4,8 @@
   "version": "1.0.0",
   "private": true,
   "description": "Preact extensions for development",
-  "main": "dist/debug.js",
   "module": "dist/debug.mjs",
+  "main": "dist/debug.js",
```

Comment:
This change seems unnecessary :upside_down_face:

Reply:
Oh, sorry that's a residual from me eliminating module :)